src-v2/NTS.Domain.Core/Objects/Startlist.cs

```diff
@@ -5,49 +5,38 @@ namespace NTS.Domain.Core.Objects;
 
 public class StartList
 {
-    public StartList() { }
+    static readonly TimeSpan START_EXPIRY_TIME = TimeSpan.FromMinutes(15);
 
-    public List<Start> Upcoming { get; set; } = new List<Start>();
-    public List<Start> History { get; set; } = new List<Start>();
+    public StartList() { }
 
-    public void OrderHistoryByAscending()
-    {
-        History = History.OrderBy(s => s.StartAt).ToList();
-    }
+    public List<Start> Starts { get; set; } = [];
 
-    public void Replace(Start start)
+    public List<Start> History
```

Comment:
Properties of the domain should not allow external actors to modify it's state unless part of the design. While the implementation of this getter currently does not allow due to the use of `ToList` (which is probably accidental, am I right) it's still not telegraphed well - it should be a `ReadOnlyCollection` or better `IReadOnlyList`.